packages/foam-vscode/package.json

```diff
@@ -388,12 +390,10 @@
     "@types/vscode": "^1.47.1",
     "@typescript-eslint/eslint-plugin": "^2.30.0",
     "@typescript-eslint/parser": "^2.30.0",
-    "babel-jest": "^26.2.2",
     "eslint": "^6.8.0",
     "eslint-plugin-import": "^2.24.2",
     "husky": "^4.2.5",
     "jest": "^26.2.2",
-    "jest-environment-vscode": "^1.0.0",
```

Comment:
This was actually doing next to nothing, it just sets `globals.vscode` so I moved that to our environment (see below) and removed the dependency (moreover, it's an obscure dependency that doesn't has a public repo).